rexpr.c

```diff
@@ -1573,7 +1574,7 @@ again:;
 		return F->t;
 
 	/* if cant get address, dont */
-	if (tmpvs [0] = R.tmpv && (!path ? !throuptr : r == 1))
+	if ((tmpvs [0] = R.tmpv && (!path ? !throuptr : r == 1)))
```

Comment:
did you check there is no bug? I mean, perhaps here should be `tmpvs[0] == R.tmv`.
I don't know really, I didn't read this code.

Reply:
Again you seem to be right here and I did a mistake, thank you !
I have applied this fixes here https://github.com/mingodad/lwc/commit/2e07cd9597ea8d0a799197c53b573df9892964a8 and here https://github.com/mingodad/pyvm/commit/0f5117ec5132006a6047813aef7e59b4d34d5377